usr/sbin/p11sak/p11sak.c

```diff
@@ -1588,7 +1606,7 @@ static const struct p11sak_opt p11sak_copy_key_opts[] = {
       .arg =  { .type = ARG_TYPE_STRING, .required = true,
                 .value.string = &opt_new_attr, .name = "ATTRS", },
       .description = "The boolean attributes to set for the copied key "
-                     "(optional):\n P L M B Y R E D G C V O W U S A X N T I. "
+                     "(optional):\n P M B Y R E D G C V O W U S X I K. "
```

Comment:
Whats the reason for this? I can somehow understand why you removed `A` and `N`, but `T` should stay there. When logged in as SO user `T` can be set.

Reply:
Please also check if the man page needs an update, too.

Reply:
Ah, this one's wrong - it was a change I'd cherry-picked from master at the time you'd just updated there. Because AIX was still on v3.21 at the time, it didn't support logging in as the SO.

I'll drop this hunk.